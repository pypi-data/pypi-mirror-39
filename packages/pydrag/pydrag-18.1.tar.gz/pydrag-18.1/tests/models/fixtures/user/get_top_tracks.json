{
    "interactions": [
        {
            "request": {
                "body": null,
                "headers": {
                    "Accept": [
                        "*/*"
                    ],
                    "Accept-Encoding": [
                        "gzip, deflate"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "User-Agent": [
                        "python-requests/2.20.0"
                    ]
                },
                "method": "GET",
                "uri": "https://ws.audioscrobbler.com/2.0/?api_key=LAST_FM_API_KEY&format=json&limit=1&method=user.getTopTracks&page=1&period=7day&user=rj"
            },
            "response": {
                "body": {
                    "string": "{\"toptracks\":{\"track\":[{\"name\":\"Einschlafhilfe Baby Sleepy - 60 Minuten\",\"duration\":\"0\",\"playcount\":\"35\",\"mbid\":\"\",\"url\":\"https://www.last.fm/music/Womb+Sounds+Heartbeat/_/Einschlafhilfe+Baby+Sleepy+-+60+Minuten\",\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"Womb Sounds Heartbeat\",\"mbid\":\"\",\"url\":\"https://www.last.fm/music/Womb+Sounds+Heartbeat\"},\"image\":[{\"#text\":\"\",\"size\":\"small\"},{\"#text\":\"\",\"size\":\"medium\"},{\"#text\":\"\",\"size\":\"large\"},{\"#text\":\"\",\"size\":\"extralarge\"}],\"@attr\":{\"rank\":\"1\"}}],\"@attr\":{\"user\":\"RJ\",\"page\":\"1\",\"perPage\":\"1\",\"totalPages\":\"64\",\"total\":\"64\"}}}"
                },
                "headers": {
                    "Access-Control-Allow-Methods": [
                        "POST, GET, OPTIONS"
                    ],
                    "Access-Control-Allow-Origin": [
                        "*"
                    ],
                    "Access-Control-Max-Age": [
                        "86400"
                    ],
                    "Access-Control-Request-Headers": [
                        "Origin, X-Atmosphere-tracking-id, X-Atmosphere-Framework, X-Cache-Date, Content-Type, X-Atmosphere-Transport, *"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Length": [
                        "594"
                    ],
                    "Content-Type": [
                        "application/json; charset=UTF-8"
                    ],
                    "Date": [
                        "Sat, 10 Nov 2018 11:28:16 GMT"
                    ],
                    "Server": [
                        "openresty/1.9.7.5"
                    ]
                },
                "status": {
                    "code": 200,
                    "message": "OK"
                }
            }
        }
    ],
    "version": 1
}
