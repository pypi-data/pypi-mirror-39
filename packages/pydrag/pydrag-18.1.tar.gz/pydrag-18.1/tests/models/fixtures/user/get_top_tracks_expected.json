{
    "data": [
        {
            "artist": {
                "mbid": "",
                "name": "Womb Sounds Heartbeat",
                "url": "https://www.last.fm/music/Womb+Sounds+Heartbeat"
            },
            "duration": 0,
            "image": [
                {
                    "size": "small",
                    "text": ""
                },
                {
                    "size": "medium",
                    "text": ""
                },
                {
                    "size": "large",
                    "text": ""
                },
                {
                    "size": "extralarge",
                    "text": ""
                }
            ],
            "mbid": "",
            "name": "Einschlafhilfe Baby Sleepy - 60 Minuten",
            "playcount": 35,
            "rank": 1,
            "url": "https://www.last.fm/music/Womb+Sounds+Heartbeat/_/Einschlafhilfe+Baby+Sleepy+-+60+Minuten"
        }
    ],
    "limit": 1,
    "page": 1,
    "total": 64,
    "user": "RJ"
}
